gatsby-theme-code-notes/gatsby-node.js

```diff
@@ -203,17 +203,6 @@ exports.createSchemaCustomization = ({ actions }) => {
       openSearch: OpenSearch
       showDate: Boolean
     }
-    type MdxFrontmatter @infer {
-      title: String!
-      tags: [String]
-      emoji: String
-      link: String
-      created: Date
-      modified: Date
-    }
-    type MdxFields @infer {
-      slug: String
-    }
```

Comment:
For some reason this broke after the upgrade to v3. I would like to add it back but after many hours of frustration, I was not able to get it to work.